modules/Vtiger/ExecuteFunctions.php

```diff
@@ -95,6 +95,29 @@
 			}
 		}
 		break;
+	case 'getFieldValuesFromMap':
+		$sourceModule = $adb->sql_escape_string(vtlib_purify($_REQUEST['sourceModule']));
+		$inputVal = $adb->sql_escape_string(vtlib_purify($_REQUEST['inputVal']));
+		$fieldname = $adb->sql_escape_string(vtlib_purify($_REQUEST['fieldname']));
+		if (!empty($inputVal)) {
+			$bmapname = $sourceModule . '_FieldInfo';
+			$cbMapid = GlobalVariable::getVariable('BusinessMapping_FieldInfo', cbMap::getMapIdByName($bmapname), $sourceModule, $current_user->id);
```

Comment:
the GV name here must be 'BusinessMapping_'.$bmapname

`GlobalVariable::getVariable('BusinessMapping_'.$bmapname, cbMap::getMapIdByName($bmapname),`

Reply:
sure